dateutil/test/test_parser.py

```diff
@@ -125,11 +127,23 @@ def testDateCommandFormatWithLong(self):
                                    tzinfos={"BRST": long(-10800)}),
                              datetime(2003, 9, 25, 10, 36, 28,
                                       tzinfo=self.brsttz))
+
     def testDateCommandFormatIgnoreTz(self):
         self.assertEqual(parse("Thu Sep 25 10:36:28 BRST 2003",
                                ignoretz=True),
                          datetime(2003, 9, 25, 10, 36, 28))
 
+    def testDateCommandFormatRespectTz(self):
```

Comment:
Ah, thanks for the pointer here. I made the test choose the right context manager and timezone name by platform.